specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2022-11-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,1971 @@
+        },
+        "secondaryKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current secondary key that clients can use to authenticate"
+        }
+      }
+    },
+    "RegenerateKeyParameters": {
+      "title": "Regenerate access keys request",
+      "type": "object",
+      "properties": {
+        "keyType": {
+          "type": "string",
+          "description": "Which access key to regenerate.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AccessKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyType"
+      ],
+      "description": "Specifies which access keys to reset to a new random value."
+    },
+    "ImportClusterParameters": {
+      "title": "Import RDB files into a target database",
+      "type": "object",
+      "properties": {
+        "sasUris": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "SAS URIs for the target blobs to import from",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUris"
+      ],
+      "description": "Parameters for a Redis Enterprise import operation."
+    },
+    "ExportClusterParameters": {
+      "title": "Export an RDB file into a target database",
+      "type": "object",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "SAS URI for the target directory to export to",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise export operation."
+    },
+    "OperationStatus": {
+      "description": "The status of a long-running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation's unique id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation's name."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the operation."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the operation."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse",
+          "description": "Error response describing why the operation failed."
+        }
+      }
+    },
+    "RegionSkuDetails": {
+      "title": "Region Sku Details",
+      "description": "List of details about all the available SKUs",
+      "type": "object",
+      "readOnly": true,
```

Comment:
> "readOnly": true, [](http://example.com/codeflow?start=5&length=18)

I'm not sure readonly at this level is meaningful, I thought it only works on properties.